src/observer.ts

```diff
@@ -9,6 +9,10 @@ import { FunctionOperator } from './operators';
 import DataLayerTarget from './target';
 import MonitorFactory from './monitor-factory';
 import { errorType, Telemetry, telemetryType } from './utils/telemetry';
+import TrackEventOperator from './operators/fsApi/trackEvent';
```

Comment:
nit: these can be exported from the `index.ts` in `/operators` and imported as a single statement.